test/integration/rules/aria-required-parent/aria-required-parent.html

```diff
@@ -1,16 +1,51 @@
-<div role="list" id="pass1"><div role="listitem" id="pass2">Item 1</div></div>
+<div role="list" id="pass1">
+	<div role="listitem" id="pass2">Item 1</div>
+</div>
+
 <div role="list" id="pass3" aria-owns="pass4"></div>
+
 <div role="listitem" id="pass4"></div>
+
 <div role="listitem" id="fail1"></div>
-<div role="menu" id="pass5"><div role="listitem" id="fail2"></div></div>
+
+<div role="menu" id="pass5">
+	<div role="listitem" id="fail2"></div>
+</div>
+
 <div role="menu" id="pass6" aria-owns="fail3"></div>
+
 <div role="listitem" id="fail3"></div>
+
 <div role="list" id="pass7" aria-owns="parent"></div>
-<div id="parent"><div role="listitem" id="pass8"></div></div>
-<div role="grid" id="pass9"><div role="rowgroup" id="pass10">Item 1</div></div>
+
+<div id="parent">
+	<div role="listitem" id="pass8"></div>
```

Comment:
I don't think this should pass. aria-owns needs to point directly to the owned element, not one of its descendants. Can you test this to make sure?